Makefile

```diff
@@ -570,6 +570,14 @@ localintegration: test-binaries ginkgo
 .PHONY: remoteintegration
 remoteintegration: test-binaries ginkgo-remote
 
+.PHONY: localbenchmarks
+localbenchmarks: test-binaries
+	ACK_GINKGO_RC=true $(GOBIN)/ginkgo \
+		      -focus "Podman Benchmark Suite" \
+		      -tags "$(BUILDTAGS) benchmarks" -noColor \
+		      -noisySkippings=false -noisyPendings=false \
+		      test/e2e/. HACK=hack/.
```

Comment:
`HACK=hack/.` seems like a NOP here, very likely a copy-paste error. Possibly harmless, but misdirecting and time-wasting.